.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+max_line_length = 120
+tab_width = 4
+trim_trailing_whitespace = true
+
+[{*.json,*.sh,*.yml}]
+indent_size = 2
+tab_width = 2
```

Comment:
Can you add `*.xml` and `*.yaml` too? I find that XML looks better with 2 spaces of indentation. This is also what `sortpom` uses.

Reply:
So, I can add `.yaml`, but so far, the files all use `.yml`. I am not sure if the 3-letter or 4-letter extension is preferred over the other.

About `.xml`, `pom.xml` was currently set to 4 spaces. I like the sortpom plugin, too, and it looks like 2 is the canonical number anyway.

It's easier, I guess, to just change the default to 2, and set 4 for .java, then.

Reply:
OK, [6bf3a95](https://github.com/package-url/packageurl-java/commit/6bf3a95054e88eeeb3861b2aa72d28d4a43484d3) just defaults to 2.